soh/src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -12939,21 +12938,25 @@ void func_8084E6D4(Player* this, PlayState* play) {
                     func_8083C0E8(this, play);
                     GameInteractor_ExecuteOnItemReceiveHooks(this->getItemEntry);
                 } else {
-                    this->actor.colChkInfo.damage = 0;
-                    func_80837C0C(play, this, 3, 0.0f, 0.0f, 0, 20);
-                    GameInteractor_ExecuteOnItemReceiveHooks(this->getItemEntry);
-                    this->getItemId = GI_NONE;
-                    this->getItemEntry = (GetItemEntry)GET_ITEM_NONE;
-                    // Gameplay stats: Increment Ice Trap count
-                    gSaveContext.sohStats.count[COUNT_ICE_TRAPS]++;
+                        this->actor.colChkInfo.damage = 0;
+                    if (!CVarGetInteger("gChestTraps", 0)) {
+                        func_80837C0C(play, this, 3, 0.0f, 0.0f, 0, 20);
+                    }
+                        GameInteractor_ExecuteOnItemReceiveHooks(this->getItemEntry);
+                        this->getItemId = GI_NONE;
+                        this->getItemEntry = (GetItemEntry)GET_ITEM_NONE;
+                        // Gameplay stats: Increment Ice Trap count
+                        gSaveContext.sohStats.count[COUNT_ICE_TRAPS]++;
```

Comment:
Many more random formatting changes above here. Kind of obfuscates what actually changes.

Reply:
this may have been me, i think i indented this section one too far inwards....